Utils.py

```diff
@@ -835,12 +834,13 @@ def run(*args: str):
         root.withdraw()
         showerror(title, text) if error else showinfo(title, text)
         root.update()
+    return None
 
 
-def title_sorted(data: typing.Iterable, key=None, ignore: typing.AbstractSet[str] = frozenset(("a", "the"))):
+def title_sorted(data: Iterable, key=None, ignore: Set[str] = frozenset(("a", "the"))):
```

Comment:
Even if the other `AbstractSet` instances are changed, this one should stay since changing it to `Set` with the `frozenset` default is a type error.

Reply:
This is not actually a type error, but it does look like one which is also not a good thing.
That's why Ruff has this rule: https://docs.astral.sh/ruff/rules/unaliased-collections-abc-set-import/
which suggests this:
```python
from collections.abc import Set as AbstractSet
```